protocols/peralta_yahya_lab/README.md

```diff
@@ -0,0 +1,46 @@
+# Flow Cytometry Preparation
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+### Partner
+Georgia Institute of Technology
```

Comment:
Should only fill in the partner section when we have actually established a partnership with lab and/or company